src/graphics/render_window.rs

```diff
@@ -328,6 +603,19 @@ impl RenderWindow {
     ///
     /// Return the position in pixels
     ///
+    /// # Usage Example
+    ///
+    /// ```no_run
+    /// # use sfml::window::{Event, Style};
+    /// # use sfml::graphics::RenderWindow;
+    /// # use sfml::system::Vector2;
+    /// # // Create a new window with SFML window as name
+    /// # let mut window = RenderWindow::new((800, 600),
+    /// #                              "SFML window",
+    /// #                              Style::CLOSE,
+    /// #                              &Default::default());
+    /// assert_eq!(window.position(), Vector2::new(0, 0));
```

Comment:
This is wrong. The window's position is not guaranteed to be `(0, 0)` when it's created.
The only reason this doesn't fail is because it's `no_run`.
Assertions in a `no_run` test are useless. In fact, worse than useless because they create misinformation, like in this case.

You should remove all assertions from `no_run` tests.

Reply:
What about set_position's example? We CANNOT run it because CI doesn't have x11, but it does work. What do you want to do about this example?

Reply:
Same for request_focus

Reply:
Same for set_size